src/content/tools/devtools/extensions.md

```diff
@@ -30,20 +34,29 @@ file in the root of the user's project
 This file stores per-project
 (or optionally, per user) settings for DevTools.
 
-If this file is **checked into source control**, the specified options are
-configured for the project. This means that anyone who pulls a project's
+If this file is **checked into source control**,
+the specified options are configured for the project.
+This means that anyone who pulls a project's
 source code and works on the project uses the same settings.
```

Comment:
**todo:** Write in active voice per [GSG](https://developers.google.com/style/voice).
```suggestion
If you **check this file into source control**,
DevTools configures the specified options for the project.
This means that anyone who pulls a project's
source code and works on the project uses the same settings.
```